api.lua

```diff
@@ -115,7 +115,7 @@ local function fire(stack, player)
 	if pointed and pointed.type == "object" then
 		local target = pointed.ref
 		local point = pointed.intersection_point
-		local dmg = base_dmg * def.dmg_mult
+		local dmg = def.base_dmg * def.dmg_mult
```

Comment:
`base_dmg` is a constant, with which `def.dmg_mult`(which is a multiplier) is multiplied to get the total damage before calculating headshots and the like. So `base_dmg` needs to be defined somewhere at the top of the file. Personally, I prefer to assign it a value of 1, meaning that each shot will deal a damage of 1 HP if `def.dmg_mult` isn't defined.